2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,9 +18,17 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let countryCurCodes = {};
+  countryCurrencyCodes.map((arr) => {
+    countryCurCodes[arr[0]] = arr[1];
+  });
```

Comment:
good job on passing the tests here - the map method works, but seeing as we don't need to create a new array, a for loop would be more appropriate